bundles/org.openhab.binding.saicismart/src/main/resources/OH-INF/config/bridge-config.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:saicismart:bridge">
+		<parameter name="username" type="text" required="true">
+			<label>Username</label>
+			<description>iSMART Username</description>
+		</parameter>
+		<parameter name="password" type="text" required="true">
+			<label>Password</label>
+			<description>iSMART Password</description>
+			<context>password</context>
+		</parameter>
+	</config-description>
```

Comment:
Is it deliberate that the SAICiSMARTBridgeConfiguration.java has a timeout property , but it is not reflected here. So it is set, but not user changable? If you change it here, it should also be added to the readme.

Reply:
I will remove this as a configuration and hard code the value in the code, as it is not for user configuration.